topic6/Main.java

```diff
@@ -0,0 +1,32 @@
+package topic6;
+
+public class Main {
+
+    public static void main(String[] args) {
+        ContulPersoanei viorel = new ContulPersoanei();
```

Comment:
Inca o sugestie ar fie sa transmiti valorile in obiect care nu ar trebui sa fie modificate in constructor, ID si nume in principiu nu se modifica intr-un sistem